src/lib/codegenerator/generator.js

```diff
@@ -33,6 +34,28 @@ export default function (templateObject = { children: [] }) {
   }
 
   counter = -1
+  isDev = this.isDev
+  if (isDev === true) {
+    ctx.renderCode.push(`
+      function propInComponent(prop, kind="dynamic") {
+        const property = prop.includes('.') ? prop.split('.')[0] : prop
+        if (kind === 'reactive' || prop.includes('.') === false) {
+          if (property in component === false) {
+            console.warn('Property ' +  property + ' was accessed during render but is not defined on instance')
```

Comment:
Is there a way we can use the blits log functionality here, instead of console.log ?